### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by aviflax**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,36 +176,31 @@
 
 <header id="top">
     <h1 id="name">Avi Flax</h1>
-    <h2 id="tagline">Developer&thinsp;/&thinsp;Designer<a href="#footnote-1" class="footnote_reference"><sup>1</sup></a></sup>&thinsp;/&thinsp;Documentarian<a href="#footnote-2" class="footnote_reference"><sup>2</sup></a></sup>&thinsp;/&thinsp;Generalist&thinsp;/&thinsp;Problem&nbsp;Solver&thinsp;/&thinsp;Software&nbsp;Human</h2>
+    <h2 id="tagline">Software Developer</h2>
+    <p id="what-i-do">I design<a href="#footnote-1" class="footnote_reference"><sup>1</sup></a>, build, document, and operate software systems.</p>
```
Comment (simadown) [inline index.html]: I appreciate the simplified language here. I also like that you've immediately taken up your active voice, moving from "Designer" to "I Design" for example.

File: index.html [hunk 2]
```diff
@@ -176,36 +176,31 @@
...
+        <li>diversity and inclusion are crucial <i>and</i> create rich, healthy, sustainable, productive organizations.</li>
         <li><a href="https://en.wikipedia.org/wiki/Don%27t_repeat_yourself">DRY</a> and <a href="https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)">the Rule of Three</a> apply to docs just as much as code.</li>
         <li>people can only do great work if they are healthy, growing, supported, and respected.</li>
         <li>all great processes have one thing in common: they are regularly reviewed and refined.</li>
-        <li>the only thing that matters about any technology is its potential to solve problems.</li>
+        <li>the most important aspect of any technology is its potential to solve problems.</li>
         <li>technology is a phenomenally powerful tool that we humans have at our disposal to solve problems — if we so choose.</li>
-        <li>we humans do some of our best thinking while <i>communicating</i> — so let’s do more of that.</li>
+        <li>we do some of our best thinking in collaboration with others.</li>
```
Comment (simadown) [inline index.html]: These are small, important tweaks that help communicate your values and how passionate you are about them without making the reader feel judged or shoulded on (you should do this, you should feel that).

File: index.html [hunk 3]
```diff
@@ -493,14 +488,12 @@ <h3>1997–1998: PCC Internet Design</h3>
     </ul>
 </details>
 
-<section id="thanks">
-    <h2>Thanks!</h2>
+<section id="contact">
+    <h2>Contact Me</h2>
     <p>Thanks for scrolling all the way to the bottom of my résumé! I appreciate your interest. At this point, you might want to get in touch. Please do!</p>
     <ul class="contact">
         <li><a href="mailto:avi@aviflax.com?subject=Job%20Offer">avi@aviflax.com</a></li>
```
[the remark this answers is not in the document]
Comment (simadown) [reply index.html]: I'm in the camp that sees this as a cute invitation but something like "I appreciate your interest and would love to hear from you" would eliminate the risk of misinterpretation that @ilangoodman brought up.

File: index.html [hunk 4]
```diff
@@ -493,14 +488,12 @@ <h3>1997–1998: PCC Internet Design</h3>
...
-        <li><a id="tel" href="tel:+1-347-426-7814">+1-347-426-7814</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/White_Plains,_New_York">White Plains, New York</a></li>
-        <li><a href="https://calendly.com/aviflax">schedule a chat</a></li>
+        <li>I’m based in <a href="https://en.wikipedia.org/wiki/White_Plains,_New_York">White Plains, New York</a> but prefer to work as part of a mostly or fully distributed team.</li>
```
Comment (simadown) [inline index.html]: Thumbs up on this change! I'm a big proponent of distributed teams but many teams and hiring managers aren't bought in (yet). Transparency here is key and will hopefully head off some difficult conversations.

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (ilangoodman) [reply index.html]: It's really just pre-populating the subject line with "Job Offer" that I think could be taken the wrong way.

[the remark this answers is not in the document]
Comment (simadown) [reply index.html]: Wow I totally missed that part!

[the remark this answers is not in the document]
Comment (aviflax) [reply index.html]: Thanks y’all! I’ll change the subject to “Interview request” :trollface:

File: index.html [hunk 5]
```diff
@@ -521,13 +514,12 @@ <h2>Notes</h2>
         <li id="footnote-1-2">Steve Jobs via <a href="https://www.nytimes.com/2003/11/30/magazine/the-guts-of-a-new-machine.html">The New York Times Magazine</a> via <a href="https://www.wired.com/2006/10/straight-dope-on-the-ipods-birth/">Wired</a></li>
```
[the remark this answers is not in the document]
Comment (aviflax) [reply index.html]: Yeah, I agree; they’re twee. I’ll figure something out.

**Comment by aviflax** [comment]
Thanks @simadown and @ilangoodman for the excellent feedback!

Responding to Ilan’s general comments:

> It's very long! I think you should consider making a short version as well.

I agree. I had hoped that the `<details>` elements (the collapsed sections) would help with this — and I think to some degree, they do. But they don’t help with the printed version of the page. I’m going to think about this. Thanks!

> * I _love_ the values section, but I think it's worth considering whether the section order is serving you well:

Great points — thanks! I’ll make some changes to address these.

> I guess overall, I'm worried that someone who doesn't read far enough would get the sense that you're a generalist who doesn't get too deep in implementing anything.. sort of a manager who doesn't like to manage.. 

I share this concern. I wonder if perhaps the values section is intrinsically problematic in this regard. Maybe moving it down will help. I do think the order of the sections telegraphs a lot to potential employers.

> …whereas what you want to convey is you're a _highly skilled_ software developer who thinks deeply and holistically about complex systems, and is able to devise _and_ implement solutions to tough problems.

🤗

**Comment by aviflax** [comment]
I believe I’ve addressed most of the feedback via 4e588c199cf65e6c3b38f41b753ecdca01e58d53 — thanks again @ilangoodman and @simadown!

**Comment by aviflax** [comment]
Deployed! New version can be seen at http://aviflax.com/resume/ — but since that website doesn’t have any versioned URLs, here’s a screenshot:

78217BAA-E00B-4139-B283-1829A955103F

----

Thanks again!

**Pull request merged**